docs/template/public/less/_side-nav.less

```diff
@@ -151,26 +151,31 @@ body.has-left-nav .sidebar-nav {
     body.has-left-nav #kss-wrapper {
         padding-left: 265px;
     }
+    .main-content {
+        position: relative; //this is not ideal
```

Comment:
I removed the comment. The CSS format was already that way. I suspect it was copied from the KSS default styles and modified. Thanks!